src/index.html

```diff
@@ -17,11 +18,368 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/index.css"
     />
   </head>
+  <!-- #endregion -->
 
   <body>
-    <h1>Catalog</h1>
+    <!-- #region header -->
+    <header class="header">
+      <div class="header__logo">
+        <a href="/">
+          <img
+            src="../src/images/logo.png"
+            alt="Site logo"
+          />
+        </a>
+      </div>
+      <nav class="header__navigation">
+        <ul class="header__list">
+          <li class="header__item">
+            <a
+              class="header__link header__link--active is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              class="header__link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              class="header__link"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li
+            class="header__item"
+            data-qa="nav-hover"
```

Comment:
```suggestion
              href="#"
            >
              Samsung
            </a>
          </li>
          
          <li class="header__item">
            <a
              class="header__link"
              href="#"
            >
              Smartphones
            </a>
          </li>
          
          <li
            class="header__item"
            data-qa="nav-hover"
```
![image](https://github.com/user-attachments/assets/df7a617d-69cc-41c2-8a30-064c75803321)

Reply:
Fix it in all cases